css/main.css

```diff
@@ -1,15 +1,17 @@
 :root {
   --color-primary: #2c72ad;
-  --color-secondary: #2f5fffdc;
+  --color-secondary: rgb(53, 144, 219);
```

Comment:
These should be consistent - either use all hex or all RGB.

Reply:
Sure!